tests/dummy/app/templates/docs/guides/upgrading.md

```diff
@@ -0,0 +1,126 @@
+
+```
+"config": {
+  "forge": "./electron-forge-config.js"
+}
+```
+
+Either way, you will need to update the configuration since the format has changed somewhat since the version of `electron-forge` that `ember-electron` 2.x used. Comparing your old configuration with the vanilla generated configuration in `electron-app/package.json` should help, and `electron-forge`'s [configuration documentation](https://www.electronforge.io/configuration) will help you through the process. Note also that the version of `electron-packager` uses changed from v11 to v14, and various maker packages were also updated, so there may be other configuration changes needed to be able to package/make with `ember-electron` 3.x.
+
+### main.js
+
+The equivalent of `ember-electron/main.js` is `electron-app/src/index.js`, so you'll want to put the contents of your `main.js` there (note that `src/index.js` is specified by the `main` entry in `electron-app/package.json`, so you can name it whatever/put it wherever you want as long as you update the `main` entry accordingy). Because the Electron project structure differs a little from `ember-electron` 2.x's, you'll need to replace anywhere you reference the path to the Ember application directory (`../ember`) with the new path (`../ember-dist`). For example, `ember-electron` 2.x's default `main.js` contains:
+
+```javascript
+protocolServe({
+  cwd: join(__dirname || resolve(dirname('')), '..', 'ember'),
+  app,
+  protocol,
+});
+```
+
+while `ember-electron` 3.x's contains:
+
+```
+protocolServe({
+  cwd: join(__dirname || resolve(dirname('')), '..', 'ember-dist'),
+  app,
+  protocol,
+});
+```
+
+### test-main.js
+
+Generally you should not need to migrate `test-main.js` at all. It's default content in 2.x was:
+
+```javascript
+/* eslint-env node */
+require('ember-electron/lib/test-support/test-main');
+```
+
+If yours still looks like that, then you don't need to do anything. If you've customized it, then you'll need to migrate the content to `electron-app/tests/index.js`
+
+### resources-* folders
+
+If you don't have any files in any of the `ember-electron/resources` or `ember-electron/resources-*` folders, you can skip this section.
+
+`ember-electron` 2.x managed the `resources-*` folders to allow you to specify platform-specific resources. `electron-forge` does not support this functionality, and it was one of the main factors contributing to the complexity and slowness of `ember-electron` 2.x's build pipeline, so it's been removed in 3.x.
+
+So if you only have content in `ember-electron/resources/`, you can just copy the folder into `electron-app`. Note that `src/index.js` is now in a subfolder, unlike `main.js` was in 2.x, so if you are accessing resources from your main process using, e.g., `path.join(__dirname, 'resources')`, you'll have to update it to `path.join(__dirname, '..', 'resources')` (or you could put the `resources` folder in the `src/` directory if you're some kind of monster).
```

Comment:
```suggestion
So if you only have content in `ember-electron/resources/`, you can copy the folder into `electron-app`. Note that `src/index.js` is now in a subfolder, unlike `main.js` was in 2.x, so if you are accessing resources from your main process using, e.g., `path.join(__dirname, 'resources')`, you'll have to update it to `path.join(__dirname, '..', 'resources')`. Alternatively, you could put the `resources` folder in the `src/` directory, but it is not recommended.
```

Reply:
The language here is trying to emphasize that this is all you need to do as far as copying files if you don't have any platform-specific resources, but if you do have platform-specific resources, you need to do some other stuff.

Maybe

> So if you only have content in `ember-electron/resources/`, you can copy the folder into `electron-app` and ignore the other `ember-electron/resources-*` folders.

?

Reply:
Seems fine to me. 👍